src/recoil/selectors/selectors.ts

```diff
@@ -0,0 +1,32 @@
+import { selector } from "recoil";
+import { getCartItems } from "../../apis";
+import { CartItemType } from "../../types";
+import { isSelectedState } from "../atoms/atoms";
+
+export const cartItemsState = selector<CartItemType[]>({
+  key: "cartItemsState",
+  get: async () => {
+    const cartItems = await getCartItems();
+
+    return cartItems;
+  },
+});
+
+export const cartPriceState = selector({
+  key: "cartPriceState",
+  get: ({ get }) => {
+    const cartItems = get(cartItemsState);
+    const isSelected = get(isSelectedState);
+
+    const orderPrice = cartItems.reduce((acc, cartItem) => {
+      if (isSelected[cartItem.id.toString()]) {
+        return acc + cartItem.product.price * cartItem.quantity;
+      }
+      return acc;
+    }, 0);
+
+    const deliveryFee = orderPrice === 0 ? 0 : orderPrice >= 100000 ? 0 : 3000;
```

Comment:
큰 숫자는 Numeric separators를 사용해서 읽기 좋게 만들어보셔도 좋을거 같아요!
```suggestion
    const deliveryFee = orderPrice === 0 ? 0 : orderPrice >= 100_000 ? 0 : 3000;
```

Reply:
삼항연산자가 중첩으로 들어가 코드를 읽기 좋지 않은데 읽기 좋은 형태로 변경해보시면 좋을거 같네요!